modules/admin-ui-picard/app/src/components/shared/wizard/WizardStepperEvent.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import {useTranslation} from "react-i18next";
+import cn from "classnames";
+import { Step, StepButton, StepLabel, Stepper } from '@material-ui/core';
+import { useStepperStyle } from '../../../utils/wizardUtils';
+import CustomStepIcon from './CustomStepIcon';
+import { checkAcls } from '../../../thunks/aclThunks';
+import { connect } from 'react-redux';
+import { checkConflicts } from '../../../thunks/eventThunks';
+
+
+const WizardStepperEvent = ({ steps, page, setPage, formik, checkAcls, checkConflicts,
+  completed, setCompleted }) => {
+  const { t } = useTranslation();
+
+  const classes = useStepperStyle();
+
+
+
+  const handleOnClick = async key => {
+    if (steps[page].name === "source") {
+      let dateCheck = await checkConflicts(formik.values);
+      if (!dateCheck) {
+        return;
+      }
+    }
+
+    if (steps[page].name === "processing" && !formik.values.processingWorkflow) {
+      return;
+    }
+
+    let aclCheck =  await checkAcls(formik.values.acls);
```

Comment:
remove one of the spaces before await
```suggestion
    let aclCheck = await checkAcls(formik.values.acls);
```